### DISCUSSION TITLE
Suggestion: Fonts and text variations

### DISCUSSION THREAD

**Discussion by bobskon** [no-chris/chord-chart-studio#654] [Ideas]
Hi, I just got started with chord chart studio. It absolutely fabulous. One thing I might suggest would be a few font choices for printing (Arial perhaps?). Another thing would be that ability to bold the chords, and increase the chord size separately from the lyrics.

Thanks for creating such a useful tool!

**Comment by no-chris** [comment]
Hello
For now ChordMark exclusively works with fixed sized fonts (monospace) and switching to another type of font such Arial would break the chords/lyrics alignment. Same with increasing the chord symbols size.
I understand this is a big limitation and I already started thinking how to solve it but unfortunately I cannot foresee short-terms improvement in this area.
Thanks for your kind comments
Christophe

**Comment by bobskon** [comment]
I figured as much. Thought I’d ask. Thanks for replying.Sent from my iPhoneOn Oct 15, 2024, at 4:07 PM, Christophe Noël ***@***.***> wrote:﻿
Hello
For now ChordMark exclusively works with fixed sized fonts (monospace) and switching to another type of font such Arial would break the chords/lyrics alignment. Same with increasing the chord symbols size.
I understand this is a big limitation and I already started thinking how to solve it but unfortunately I cannot foresee short-terms improvement in this area.
Thanks for your kind comments
Christophe

**Comment by karelbasta** [comment]
Hi,

I understand the problem with monospace font, but could you at least add the option to make Chords **Bold** when exported to PDF?

![image](https://github.com/user-attachments/assets/b5bf2caf-1cef-471c-b35e-367ec5e0824c)

It would make it easier for me to distinguish between lyrics and chords that way.
